index.html

```diff
@@ -18,16 +18,145 @@
 
     <!-- font awesome -->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css"/>
+    <style>
+      .player {
+          height: 92vh;
+          padding: 0;
+          background-color: none;
+        }
+
+        .screen {
+          height: 68vh;
+          background-color: none
+        }
+
+        #myCanvas {
+          border: 1px solid #000000
+        }
+
+        /* .toolbox {
+          background-color:#1D3557;
+          color: #fdfdfd
+        } */
+
+        .toolbox-basic {
+          height: 10vh;
+          width: 100%;
+        }
+
+        .play-button {
+          background-color: none;
+        }
+
+        .player-slider {
+          min-width: 30vw;
+        }
+
+        .extra-tools {
+          height: 12vh;
+        }
+
+        .screen-canvas {
+          /* max-width: 50vw;
+          max-height: 50vh; */
+        }
+
+        .scroll-sect {
+          overflow-y: scroll; 
+          height: 10vh;
+        }
+
+        
+
+    </style>
   </head>
 
   <body>
     <v-app class="window" id="app">
       <div class="row no-gutters">
-        <navbar @canvas-changed="changeCanvas" class="col-12"></navbar>
+        <navbar @canvas-changed="changeCanvas" @file-changed="changeFile" class="col-12"></navbar>
 
-        <leftpanel @layer-selected="layerSelected" class="col-3"></leftpanel>
+        <leftpanel @layer-selected="layerSelected" :layers="layers" :key="componentKey" class="col-3"></leftpanel>
+        
+        <!-- Player  -->
+        <div class="col-6 bg-white">
+          <div class="player d-flex flex-column justify-content-between">
+            <!-- <p>{{ canvasStyling[canvasShape] }}</p> -->
```

Comment:
Remove. I need a bot :-/

Reply:
:( We will erase all of these unneeded codes.